pkgs/development/python-modules/pygame_sdl2/default.nix

```diff
@@ -4,12 +4,12 @@
 buildPythonPackage rec {
   pname = "pygame_sdl2";
   version = "2.1.0";
-  renpy_version = "7.2.0";
+  renpy_version = "8.0.0";
   name = "${pname}-${version}-${renpy_version}";
 
   src = fetchurl {
     url = "https://www.renpy.org/dl/${renpy_version}/pygame_sdl2-${version}-for-renpy-${renpy_version}.tar.gz";
-    sha256 = "1amgsb6mm8ssf7vdcs5dr8rlxrgyhh29m4i573z1cw61ynd7vgcw";
+    sha256 = "0majf64pdfba5byjlv41pgsdmwvy09hw3m7143jz3kc1wjd2gaw8";
```

Comment:
Could you change this to SRI style hash?

Reply:
Sure.